assessment.rb

```diff
@@ -1,12 +1,32 @@
 require 'csv'
 # Assessment answers to:
+rails g model Licensee surname:string given_name:string gender:string height:integer weight:integer dob:date,eye_color:string hair_color:string
 
 # question 2:
+caren = Licensee.create surname: 'Bishop', given_name: 'Caren', gender: 'f', height:63, wieght:132, dob:1943-09-26, eye_color:'Brown', hair_color:'Black'
 
 # question 3:
+rails g model CreateLicensees
+require 'data/licensees.csv'
+class CreateCreateLicensees < ActiveRecord::Migration
+  def change
+    create_table :create_licensees do |t|
 
-# question 4:
+      t.timestamps null: false
+    end
+  end
+end
 
+#then do something but I don't know what and I can't find it.
+# question 4:
+Licensee.find_by! eye_color:'Brown' and hair_color:'Brown' or hair_color:'Black'
```

Comment:
Not correct syntax. There are a couple of ways to do this one, since `find_by`/`where` accept a couple of different input formats. 

Here's one possible answer. Can you figure out any others?
`Licensee.find_by! eye_color:'Brown', hair_color:['Brown', 'Black']`